server/text_generation_server/models/causal_lm.py

```diff
@@ -850,9 +850,12 @@ def forward(
             "attention_mask": attention_mask,
             "past_key_values": past_key_values,
             "token_idx": token_idx,
-            "lazy_mode": LAZY_MODE == 1,
         }
 
+        # Optimum Habana got lazy_mode only supported for llama type of models
```

Comment:
Hmm rather torch.compile that is only supported for Llama models no?

Reply:
I meant that "lazy_mode" attribute is only supported for a type of models "LLAMA"